tests/docker/php/cli/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM php:8.1-cli
+
+MAINTAINER  A. Egorov <anat-gor@ya.ru>
+
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
+RUN chmod +x /usr/local/bin/install-php-extensions
+
+
+RUN install-php-extensions redis intl pcntl xdebug @composer
```

```suggestion
RUN pecl install redis && docker-php-ext-enable redis intl pcntl xdebug
```
(some of them should be installed first)

Don't see any reasons to use 3rd party libraries here for that simple task